src/util/ReactUtils.ts

```diff
@@ -97,11 +97,11 @@ export const toArray = <T extends ReactNode>(children: T | T[]): T[] => {
  * Find and return all matched children by type. `type` can be a React element class or
  * string
```

Comment:
Can we update this comment? It looks like now `type`  can only be `ComponentType | ComponentType[]` which does not include string

Also does this work with a Function component?

Reply:
> Can we update this comment?

Sure, I haven't noticed that earlier. 😅

> Also does this work with a Function component?

`ComponentType` includes both component types
```ts
type ComponentType<P = {}> = ComponentClass<P> | FunctionComponent<P>;
```